cvat/apps/lambda_manager/serializers.py

```diff
@@ -5,30 +5,42 @@
 from drf_spectacular.utils import extend_schema_serializer
 from rest_framework import serializers
 
+class AttributeMappingSerializer(serializers.JSONField):
+    def validate(self, attrs):
+        for attr_name in self.initial_data:
+            if not isinstance(self.initial_data[attr_name], str):
+                raise serializers.ValidationError(f"Not valid field value type: {attr_name}, must be string")
+        return attrs
+
+class SkeletonLabelMappingEntrySerializer(serializers.DictField):
+    name = serializers.CharField()
+    attributes = AttributeMappingSerializer(required=False)
 
 class LabelMappingEntrySerializer(serializers.DictField):
     name = serializers.CharField()
+    attributes = AttributeMappingSerializer(required=False)
+    elements = serializers.DictField(child=SkeletonLabelMappingEntrySerializer(), required=False,
+        help_text="Label mapping from the model skeleton elements to the task skeleton elements"
```

Comment:
```suggestion
    sublabels = serializers.DictField(child=SublabelMappingEntrySerializer(), required=False,
        help_text="Sublabel mapping from the model shape elements to the task shape elements"
```

1. We should follow existing terminology, where labels have sublabels and shapes have elements.
2. While currently only skeletons can have elements, this is not set in stone, so it would be better to avoid hardcoding references to skeletons.